src/ctree-app/ctree-landing.html

```diff
@@ -246,31 +376,55 @@
-              <span class="bold">contact@innovativeFuture</span>.
-            </p>
-          </section>
+          <p>W</p>
+          <p>W</p>
+          <p>W</p>
+        </div>
+        <div id="participate" class="section-styling">
+            <div class= "center-div">
+              <h1 align="center">Participate</h1>
+                <p>
+                  <div class="bold-text">Join the open source project</div>
+                  Get course credit or professional grade experience working on a large open source
+                  project.  Be part of a project team, working closely with our programmers and
+                  designers to complete a part of the project.  All regular contributors can add the
+                  project to their LinkedIn resume and request a recommendation based on their comleted
+                  work.  Go to <a href="dev.ctree.org">dev.ctree.org</a> to get started.
+                  </p>
+                  <p>
+                    <div class="bold-text">Test the project</div>
+                    As the project develops we need real world examples of how it can be used.  If you
+                    want to use it with your nonprofit, online community, or other organization Please
+                    email <a href = "mailto: contact@innovativeFuture.org">contact@innovativeFuture.org</a>
+                    </p>
+            </div>
+        </div>
+        <div id="footer" class="section-styling">
+          <div class="row">
+            <div class="col-5 table-cell line">
+              <div class="inner-row-1">
+                <div id="social-media-icon-bar">
+                  <ul>
+                    <li><a href="https://twitter.com/"><i class="twitter"></i><img src="[[rootPath]]/images/twitter-icon-32.png"></a></li>
+                    <li><a href="https://www.facebook.com/"><i class="facebook"</i><img src="[[rootPath]]/images/facebook-icon-32.png"></a></li>
+                    <li><a href="https://github.com/F4IF/ctree-demo"><i class="github"></i><img src="[[rootPath]]/images/github-icon-32.png"></a></li>
+                    </ul>
+                </div>
+              </div>
+              <div class="inner-row-2" id="ctree-blurb">
+                <p>The Collaboration Tree is a project of the Foundation For an Innovative Future</p>
+              </div>
+            </div>
+            <div class="col-7 table-cell">
+              <div id="copyright">
+                <p>
+                  Nulla facilisi. Donec felis ligula, feugiat nec imperdiet vitae, gravida quis felis. Aliquam ultrices ultrices eros, sit amet tristique dui ultrices quis. Vivamus semper mauris eget mauris lacinia, sit amet interdum neque mollis. Suspendisse potenti. Phasellus ut eros tempus, cursus nulla id, accumsan diam. Phasellus rhoncus metus sit amet mattis mollis. Donec elementum in magna ut vehicula. Lorem ipsum dolor sit amet, consectetur adipiscing elit.
```

Comment:
Based on the wireframes it looks like this section should have a grid of labels with the copyright below.  Also, on mobile there's supposed to be a horizontal line separating it from the social media icon bar.

Reply:
What should the labels contain?  It's not clear in the wireframe.

Reply:
You can just put anything (ex. test1, test2, test3, etc.).  Just add enough to showcase the grid.

Reply:
Added.